packages/pixeloven-webpack/config/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import { removeEmpty } from "@pixeloven-core/common";
 import { getUtils } from "@pixeloven-core/env";
 import CaseSensitivePathsPlugin from "case-sensitive-paths-webpack-plugin";
-import CircularDependencyPlugin from "circular-dependency-plugin";
+// import CircularDependencyPlugin from "circular-dependency-plugin";
```

Comment:
SSR has way to many of these so I need a way to only show a few at a time and then an indicator more are their.